pkg/skaffold/runner/v1/dev.go

```diff
@@ -226,6 +230,32 @@ func (r *SkaffoldRunner) Dev(ctx context.Context, out io.Writer, artifacts []*la
 			continue
 		}
 
+		var absMatches []string
+		// Add absolute typed rules
+		if artifact.Sync != nil {
+			for _, pt := range artifact.Sync.Manual {
+				if filepath.IsAbs(pt.Src) {
+					// list files and add to dependencies list
+					matches, err := doublestar.Glob(pt.Src)
+					if err != nil {
+						return sErrors.NewErrorWithStatusCode(
+							proto.ActionableErr{
+								Message: fmt.Sprintf("`%q` pattern: %s", pt.Src, err.Error()),
+								ErrCode: proto.StatusCode_INIT_SYNC_ABS_PATH_ERR,
```

Comment:
@tejal29 shouldn't this be DEVINIT as per `StatusCode_DEVINIT_REGISTER_BUILD_DEPS` and `StatusCode_DEVINIT_REGISTER_TEST_DEPS` below?